CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 [BUGFIX] Remove misleading config error message
 [BUGFIX] Re-use existing root store
 [BUGFIX] Use standard Unix directories on MacOS
+[BUGFIX] Allow secret names to have a colon in the name
```

Comment:
Thank you for remembering to add this to the changelog.

In the past we didn't enforce this so we have a CommitCheck that looks for a RELEASE_NOTES= entry.

Could you please add `RELEASE_NOTES=[BUGFIX] Allow secret names to have a colon in the name` to the commit message so we can automatically parse that when we create the next release?